cmake/Modules/FindComputeCpp.cmake

```diff
@@ -74,13 +74,13 @@ find_program(ComputeCpp_INFO_EXECUTABLE computecpp_info
   NO_SYSTEM_ENVIRONMENT_PATH)
 
 find_library(COMPUTECPP_RUNTIME_LIBRARY
-  NAMES ComputeCpp ComputeCpp_vs2015
```

Comment:
Nit: I'd be tempted to leave the old library names available, so this doesn't just break when using the old windows packages.